elastalert/config.py

```diff
@@ -133,7 +135,9 @@ def load_options(rule, conf, args=None):
     except (KeyError, TypeError) as e:
         raise EAException('Invalid time format used: %s' % (e))
 
-    # Set defaults
+    # Set defaults, copy defaults from config.yaml
+    for key, val in base_config.items():
```

Comment:
Shouldn't you do this for after the other rule.setdefault lines below? If somebody were to set their timestamp globally in config.yaml, you would overwrite it on line 147.  Is that a supported scenario to put arbitrary rule parameters like that in config.yaml?